packages/page-bounties/src/BountyCreate.tsx

```diff
@@ -27,14 +27,20 @@ function BountyCreate () {
   const [bond, setBond] = useState(((api.consts.bounties || api.consts.treasury).bountyDepositBase as BalanceOf).toBn());
   const [value, setValue] = useState<BN | undefined>(BOUNTY_DEFAULT_VALUE);
   const [isOpen, toggleIsOpen] = useToggle();
+  const [isTitleValid, setIsTitleValid] = useState(false);
+  const [isValueValid, setIsValueValid] = useState(false);
+  const [hasFunds, setHasFunds] = useState(false);
 
   const balances = useCall<DeriveBalancesAll>(api.derive.balances.all, [accountId]);
-  const bountyMinValue = ((api.consts.bounties || api.consts.treasury).bountyValueMinimum as BalanceOf).toBn();
-  const bountyTitleMaxLength = ((api.consts.bounties || api.consts.treasury).maximumReasonLength as BalanceOf).toNumber();
 
-  const isTitleValid = title?.length >= MIN_TITLE_LEN && countUtf8Bytes(title) <= bountyTitleMaxLength;
-  const isValueValid = value?.gte(bountyMinValue);
-  const hasFunds = balances?.availableBalance.gte(bond);
+  useEffect(() => {
```

Comment:
I would recommend splitting value & title validation into 2 parts, i.e. validator-for-value only happens when the value changes and validation-for-title only applies when the title changes.